File: backend/Domain/Entities/RepresentativeContact.cs
namespace Domain.Entities
{
    public class RepresentativeContact : BaseLogDomain
    {
        public int Id { get; set; }
        public string? Value { get; set; }
        public bool? AllowNotification { get; set; }
        public int TypeId { get; set; }
        public int RepresentativeId { get; set; }
    }
}

File: frontend/src/constants/RepresentativeContact.ts
import { Dayjs } from "dayjs";

export type RepresentativeContact = {
  
  id: number;
  value: string;
  allowNotification: boolean;
  rTypeId: number;
  representativeId: number;
};


export type RepresentativeContactCreateModel = {
  
  id: number;
  value: string;
  allowNotification: boolean;
  rTypeId: number;
  representativeId: number;
};
